lib/govuk_navigation_helpers/version.rb

```diff
@@ -1,3 +1,3 @@
 module GovukNavigationHelpers
-  VERSION = "6.0.1".freeze
```

Comment:
Versions shouldn't be updated in the same PR as the change itself. Also needs a changelog entry.